src/static/js/utils/process-json.js

```diff
@@ -1,53 +1,5 @@
 'use strict';
 
```

Comment:
Looks like `index.js` is still using `indexOf` on line 8. Do we want to:

a) refactor index.js to not use it or 
b) include the polyfill there or separately, OR
c) it's moot because we confirmed the code is working as expected in all supported browsers aka ie8?